homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -132,14 +129,14 @@ async def async_update(self) -> None:
             if await afsapi.get_power():
                 status = await afsapi.get_play_status()
                 self._attr_state = {
-                    PlayState.PLAYING: STATE_PLAYING,
-                    PlayState.PAUSED: STATE_PAUSED,
-                    PlayState.STOPPED: STATE_IDLE,
+                    PlayState.PLAYING: MediaPlayerState.PLAYING,
+                    PlayState.PAUSED: MediaPlayerState.PAUSED,
+                    PlayState.STOPPED: MediaPlayerState.IDLE,
                     PlayState.LOADING: STATE_OPENING,
```

Comment:
Do we allow custom states for media_player?

Reply:
I think that MediaPlayerState.BUFFERING is an approrioate analog for PlayState.LOADING